tests/ballerina-test/src/test/resources/test-src/types/integer/integer-value.bal

```diff
@@ -1,56 +1,109 @@
-function testIntegerValue() (int) {
+//
+// Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+//
+
+function testIntegerValue () (int) {
     int b;
     b = 10;
     return b;
 }
 
-function testNegativeIntegerValue() (int) {
+function testNegativeIntegerValue () (int) {
     int y;
     y = -10;
     return y;
 }
 
-function testIntegerValueAssignmentByReturnValue() (int) {
+function testHexValue () (int) {
```

Comment:
Shall we also add some test cases for doing math operations (add/sub), and logical operations (quality/greater-than/etc) for  integers of mixed types? (i.e: adding/comparing two integers, one in hex and the other in binary)